### PR TITLE
Feature/fga/more realm fixing

### PR DESCRIPTION
**Pull request by ganfra**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [X] Technical
- [ ] Other :

## Content

Continue work started by https://github.com/vector-im/element-android/pull/5297

### CODE REVIEW

File: matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/timeline/TimelineService.kt [hunk 1]
```diff
@@ -41,15 +41,15 @@ interface TimelineService {
...
+    fun getTimelineEvent(eventId: String): TimelineEvent?
 
     /**
      * Creates a LiveData of Optional TimelineEvent event with eventId.
      * If the eventId is a local echo eventId, it will make the LiveData be updated with the synced TimelineEvent when coming through the sync.
      * In this case, makes sure to use the new synced eventId from the TimelineEvent class if you want to interact, as the local echo is removed from the SDK.
      * @param eventId the eventId to listen for TimelineEvent
      */
-    fun getTimeLineEventLive(eventId: String): LiveData<Optional<TimelineEvent>>
+    fun getTimelineEventLive(eventId: String): LiveData<Optional<TimelineEvent>>
```
Comment (bmarty) [inline matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/timeline/TimelineService.kt:52]: Can you add a file `5330.sdk` to announce the 2 API name changes?

**Review by bmarty — APPROVED** [review]
LGTM, just a small request for the future changelog

File: matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/timeline/TimelineEventDataSource.kt [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
```
Comment (bmarty) [inline matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/timeline/TimelineEventDataSource.kt:2]:
```suggestion
 * Copyright (c) 2022 The Matrix.org Foundation C.I.C.
```

**Review by ouchadam — APPROVED** [review]
:crossed_fingers:

**Review by bmarty — CHANGES_REQUESTED** [review]
Blocked by https://github.com/vector-im/element-android/pull/5330#discussion_r813199337

File: matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/timeline/TimelineService.kt (hunk 1 above)
Comment (ganfra) [reply matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/timeline/TimelineService.kt:52]: done

**Review by bmarty — APPROVED** [review]
thanks!

**Pull request merged**